spec/system/admin/products_v3/update_spec.rb

```diff
@@ -706,4 +739,11 @@
       include_examples "updating image"
     end
   end
+
+  # Check a validation message is set, we don't check the message itself because the value is based
+  # on the browser's locale.
+  def expect_browser_validation(selector)
+    browser_message = page.find(selector)["validationMessage"]
+    expect(browser_message.present?).to be(true)
```

Comment:
I find the `be(true)` matcher confusing when it fails. I would either use `eq` or:
```suggestion
    expect(browser_message).to be_present
```

Reply:
I always forget about these !